services/server/package.json

```diff
@@ -79,6 +79,7 @@
     "express-rate-limit": "^7.5.0",
     "express-session": "^1.18.0",
     "jsonwebtoken": "^9.0.2",
+    "jssha": "^3.3.1",
```

Comment:
We don't need this package. We can use the crypto node provided by nodejs. It is quite simple.

```ts
import { createHash } from 'node:crypto'

function sha256(content: string) {  
  return createHash('sha256').update(content).digest('hex')
}
```